core/creator/src/main/java/io/quarkus/creator/phase/runnerjar/RunnerJarPhase.java

```diff
@@ -346,6 +340,54 @@ public void accept(Path path) {
         }
     }
 
+    /**
```

Comment:
I am not sure if the below comments are necessary..

Reply:
I do prefer to leave that comment since it explain why this method exist

Reply:
I do prefer to leave that comment since it explain why this method exist